src/core/credential_format/mod.rs

```diff
@@ -213,6 +216,10 @@ pub enum ClaimFormatDesignation {
     /// the Credential format can be utilized with any type of Credential (or mdoc document types).
     #[serde(rename = "mso_mdoc")]
     MsoMDoc,
+    /// The format is defined by IETF Web Authorization Protocol (https://datatracker.ietf.org/doc/draft-ietf-oauth-selective-disclosure-jwt/)
+    /// which defines a Selective Disclosure for JWTs (SD-JWT)
```

Comment:
The description should be more precise about the credential format. SD-JWT is just the enveloppe, and that doesn't say anything about how the credential is encoded as the enveloppe's payload (technically we just know it's JSON). What we want to target is the [VCDM2.0 JOSE COSE SD-JWT](https://w3c.github.io/vc-jose-cose/#with-sd-jwt) format. It's a VCDM2.0 credential, interpreted as JWT claims, envelopped into an SD-JWT.